README.md

```diff
@@ -2,13 +2,15 @@
 
 ![readme-banner.png](assets/readme-banner.png)
 
-[![Discord](https://img.shields.io/discord/889460117953720351?color=%237289DA&logo=discord&logoColor=white&style=for-the-badge)](https://discord.gg/FQ7jmGBd6c)
-[![Website](https://img.shields.io/badge/-website-orange?style=for-the-badge&logo=internet-explorer&logoColor=white)](https://eternalcode.pl/)
+[![Supports Paper](https://raw.githubusercontent.com/intergrav/devins-badges/v3/assets/cozy/supported/paper_vector.svg)](https://papermc.io)
+[![Supports Spigot](https://raw.githubusercontent.com/intergrav/devins-badges/v3/assets/cozy/supported/spigot_vector.svg)](https://spigotmc.org)
```

Comment:
Wszystko jest okej, jedynie z tego bym wywalił przekierowanie do stron.

Reply:
Moim zdaniem to przekierowanie to fajna sprawa, można od razu wejść na strony tych dostawców oprogramowania. Co sądzi o tym reszta zespołu?

Reply:
Spoko, to niech zostanie